### ISSUE TITLE
Missing WORKDIR line in Dockerfile

### ISSUE THREAD

**Issue by edgetrader** [sherlock-project/sherlock#643]
I think you are missing a line in your Dockerfile.

**WORKDIR /opt/sherlock/sherlock**

```
RUN pip3 install -r requirements.txt -f /wheels \
  && rm -rf /wheels \
  && rm -rf /root/.cache/pip/*

### MISSING LINE
WORKDIR /opt/sherlock/sherlock

ENTRYPOINT ["python", "sherlock.py"]
```

**Comment by sdushantha** [comment]
@edgetrader I dont use Docker, but if this fixes any issue you are having, would you like to create a PR for this change?

**Comment by sdushantha** [comment]
This has been fixed in #672
